aspnetcore/signalr/configuration.md

```diff
@@ -0,0 +1,239 @@
+| `ApplicationMaxBufferSize`  | The maximum number of bytes received from the client that the server will buffer. Increasing this value allows the server to receive larger messages, but can negatively impact memory consumption. The default value is 32KB. |
+| `AuthorizationData` | A list of [`IAuthorizeData`](/dotnet/api/microsoft.aspnetcore.authorization.iauthorizedata) objects used to determine if a client is authorized to connect to the hub. By default, this is populated with values from the `Authorize` attributes applied to the Hub class. |
+| `TransportMaxBufferSize`  | The maximum number of bytes sent by the application that the server will buffer. Increasing this value allows the server to send larger messages, but can negatively impact memory consumption. The default value is 32KB. |
+| `Transports`  | A bitmask of [`HttpTransportType`](LINK TBD, API DOCS BEING UPDATED) values that can restrict the transports a client can use to connect. By default, all transports are enabled. |
+| `LongPolling`  | Additional options specific to the Long Polling transport |
+| `WebSockets`  | Additional options specific to the WebSockets transport |
+
+The Long Polling transport has additional options that can be configured using the `LongPolling` property:
+
+| Option | Description |
+| ------ | ----------- |
+| `PollTimeout` | The maximum amount of time the server will wait for a message to send to the client before terminating a single poll request. Decreasing this value will cause the client to issue new poll requests more frequently. The default value is 90 seconds. |
+
+
+The WebSocket transport has additional options that can be configured using the `WebSockets` property:
+
+| Option | Description |
+| ------ | ----------- |
+| `CloseTimeout`  | After the server closes, if the client fails to close within this time interval, the connection will be terminated. |
+| `SubProtocolSelector` | A delegate that can be used to set the `Sec-WebSocket-Protocol` header to a custom value. The delegate receives the values requested by the client as input and is expected to return the desired value. |
+
+## Configure client options
+
+Client options can be configured on the `HubConnectionBuilder` type (available in both .NET and JavaScript clients), as well as on the `HubConnection` itself.
+
+### Configure logging
+
+Logging is configured in the .NET Client using the `ConfigureLogging` method. Logging providers and filters can be registered in the same way as they are on the server. See the [Logging in ASP.NET Core](xref:fundamentals/logging/index#how-to-add-providers) documentation for more information.
+
+> [!NOTE]
+> In order to register Logging providers, you must install the necessary packages. See the [Built-in logging providers](xref:fundamentals/logging/index#built-in-logging-providers) section of the docs for a full list.
+
+For example, to configure the console logger (after installing the `Microsoft.Extensions.Logging.Console` NuGet package):
```

Comment:
The for example sounds a bit out of place after a note and the paragraph before it.
Maybe
For example, to configure the console logger (after installing the `Microsoft.Extensions.Logging.Console` NuGet package):
->
The following example shows how to configure the console logger after installing the `Microsoft.Extensions.Logging.Console` NuGet package:

Reply:
How about:

For example, to enable Console logging, install the `Microsoft.Extensions.Logging.Console` NuGet package and call the `AddConsole` extension method: